bugtool/cmd/configuration.go

```diff
@@ -201,6 +198,19 @@ func catCommands() []string {
 	return commands
 }
 
+// routeCommands gets the routes tables dynamically.
+func routeCommands() []string {
+	commands := []string{}
+	routes, _ := execCommand("ip route show table all | grep -E --only-matching 'table [0-9]+'")
```

Comment:
@youssefazrak  does this mean we *won't* dump from tables like `table local` / `table main`? I guess `table main` is already handled by the `"ip -4 r"` case above, but could `table local` also have relevant information?

Reply:
Hey @joestringer Indeed that is my undersanding. `ip -4 r` drop the `table main` but not `table local`.
The `local` routes of the `table local` are the routes to a locally hosted IP, basically the L3 interfaces' IPs. We already have this output from the `ip a` command.
The `broadcast` part for the broadcast addresses but this is also given by `ip a`.

Reply:
OK, that sounds reasonable to me. Thanks for the clarification!